sklearn/svm/base.py

```diff
@@ -779,6 +779,12 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     if verbose:
         print('[LibLinear]', end='')
 
+    # LinearSVC breaks when intercept_scaling is 0
+    if intercept_scaling is 0:
```

Comment:
here yes. in the test, no (it is true by default). You can add an additional test that intercept_scaling is ignored when ``fit_intercept=False``, though.